web/apps/labelstudio/src/pages/Projects/ProjectsList.jsx

```diff
@@ -119,7 +121,7 @@ const ProjectCard = ({ project }) => {
         </Elem>
         <Elem name="description">{project.description}</Elem>
         <Elem name="info">
-          <Elem name="created-date">{format(new Date(project.created_at), "dd MMM ’yy, HH:mm")}</Elem>
+          <Elem name="created-date">{format(new Date(project.created_at), "dd MMM 'yy, HH:mm")}</Elem>
```

Comment:
not sure here though, but this change is also out of scope of this PR

Reply:
it's probably the formatter. i definitely didn't touch it